globals.tf

```diff
@@ -0,0 +1,18 @@
+variable "env" {
+  description = "Environment name"
+}
+
+variable "office_cidrs" {
+  description = "CSV of CIDR addresses for our office which will be trusted"
+  default     = "80.194.77.90/32,80.194.77.100/32"
+}
+
+variable "web_access_cidrs" {
+  description = "CSV of CIDR addresses for which we allow web access"
+  default     = "80.194.77.90/32,80.194.77.100/32"
+}
```

Comment:
```env``` is used to provide prefix for environment. You specify it while running terraform like this: ```terraform apply --var env=alex``` . ```office_cidrs``` are used in security group to provide ssh access. I have removed web_access_cidrs.